README.md

```diff
@@ -1,7 +1,13 @@
 # LinkFree - EddieHub
 
-An open-source alternative to [Linktree](https://linktr.ee/) implemented in JavaScript - built by the [EddieHub](https://www.eddiehub.org)\
-Initially created on a YouTube live stream <https://www.youtube.com/watch?v=Jorl_vcp-Ew>
+An open-source alternative to [Linktree](https://linktr.ee/) implemented in JavaScript - built by the [EddieHub](https://www.eddiehub.org).\
+Initially created on a [YouTube live stream](https://www.youtube.com/watch?v=Jorl_vcp-Ew).
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree)
+
+![Screenshot](https://user-images.githubusercontent.com/60853067/133296120-dbdb1799-4cca-4708-81ce-05edc65e59c9.png)
```

Comment:
- Please add the alternative text a bit relative

Reply:
Open in gitpod seems fine enough

Reply:
For the screenshot part.. "A screenshot of the running application on a profile page"